docs/absolute.md

```diff
@@ -334,6 +334,25 @@ epoch.inTimeZone(utc).difference(billion.inTimeZone(utc), { largestUnit: 'years'
   // => P31Y8M8DT1H46M40S
 ```
 
+### absolute.**equal**(_other_: Temporal.Absolute) : boolean
+
+**Parameters:**
+- `other` (`Temporal.Absolute`): Another time to compare.
+
+**Returns:** `true` if `absolute` and `other` are equal, or `false` if not.
+
+Compares two `Temporal.Absolute` objects for equality.
```

Comment:
Here and elsewhere: it would be good, via either description text or the examples, to explain why you need `.equals()` and can't just use `===`

Reply:
Very good point, will do.